crm_claim/models/crm_claim.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2015-2017 Odoo S.A.
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+from odoo.tools import html2plaintext
+
+APPLICABLE_MODELS = [
+    "account.invoice",
```

Comment:
```suggestion
    "account.move",
```
This looks like a reminiscence from the past.